src/core.js

```diff
@@ -1,3 +1,4 @@
+import * as d3 from 'd3';
```

Comment:
With the current settings of rollup, this part doesn't look up the global d3 in browser environment (This only works in commonJS and AMD environment). I think it's safe to keep that line as is.